specifyweb/frontend/js_src/lib/components/Formatters/Formatter.tsx

```diff
@@ -204,13 +194,164 @@ function Definitions({
             }
           >
             {resourcesText.addDefinition()}
-          </Button.Success>
+          </Button.Info>
         </div>
       ) : null}
     </div>
   );
 }
 
+function ConditionalFormatter({
```

Comment:
@maxpatiiuk, 
Should I create a new file with the smaller component to reduce the length of this one?

Reply:
yes please 👍